pkg/proxy/iptables/proxier.go

```diff
@@ -778,10 +901,22 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 		activeNATChains[svcChain] = true
 
+		svcLbChain := serviceLBChainName(svcName, protocol)
```

Comment:
@freehan called this KUBE-FW-*, you called it KUBE-LB-* - both are a bit unsatisfying.  How about KUBE-XLB (external LB) ?  Since you need Minhan's blackhole support, I'll hit his review next